src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,89 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int MULTIPLIER = 2;
+    private static final int INITIAL_CAPACITY = 16;
+    private int currentCapacity;
+    private int size;
+    private Node<K, V>[] elements;
+
+    public MyHashMap() {
+        this.currentCapacity = INITIAL_CAPACITY; // Инициализация начальной емкости
+        elements = (Node<K, V>[]) new Node[currentCapacity];
+        this.size = 0;
```

Comment:
[where do we need this](https://stackoverflow.com/a/2411283)
0 is default for int